knowledge_repo/app/templates/index-base.html

```diff
@@ -125,7 +125,9 @@
     } else {
       next_parameters["start"] = next_parameters["results"];
     }
-    $(".next-btn")[0].href += "?" + $.param(next_parameters);
+    if ($(".next-btn").length) {
+      $(".next-btn")[0].href += "?" + $.param(next_parameters);
```

Comment:
actually, a better thing to do rather than checking length, would be to check for existence.